src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -154,105 +189,178 @@ function AdminAddCoursesForm(props) {
+                            moment(formValues.end_time, 'HH:mm'),
+                          ]}
+                          onChange={handleTimeChange}
+                          use12Hours={true}
+                          format="HH:mm"
+                          name="course_time"
+                        />
+                        <br></br>
+                      </label>
+                    ))} */}
               </Form.Item>
               <Form.Item
                 label={
                   <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
-                    Instructor ID:
+                    Location:
                   </label>
                 }
                 style={{ width: '65%', fontSize: '1.1rem' }}
                 onChange={handleChange}
               >
-                <InputNumber
-                  value={formValues.instructor_id}
-                  name="instructor_id"
+                <Input
+                  value={formValues.course_location}
+                  name="course_location"
                   style={{ fontSize: '1.1rem' }}
                 />
               </Form.Item>
               <Form.Item
                 label={
                   <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
-                    Days of the Week:
+                    Maximum Capacity:
                   </label>
                 }
-                valuePropName="checked"
-                style={{ width: '100%', fontSize: '1.1rem' }}
-                onChange={handleCheck}
+                style={{ width: '65%', fontSize: '1.1rem' }}
+                onChange={handleChange}
               >
-                <Checkbox.Group options={daysOfWeek} />
+                <InputNumber
+                  value={formValues.max_size}
+                  name="max_size"
+                  style={{ fontSize: '1.1rem' }}
+                />
               </Form.Item>
               <Form.Item
+                style={{ width: '65%', fontSize: '1.1rem' }}
                 label={
                   <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
-                    Start and End Time:
+                    Minimum Age:
                   </label>
                 }
-                style={{ width: '60%', fontSize: '1.1rem' }}
+                onChange={handleChange}
               >
-                {formValues.course_days.map(day => (
-                  <label key={`${day}`}>
-                    {`${day}:`}
-                    <TimePicker.RangePicker
-                      onChange={handleTimeChange}
-                      use12Hours={true}
-                      format="HH:mm"
-                      name="course_time"
-                    />
-                  </label>
-                ))}
+                <InputNumber
+                  value={formValues.min_age}
+                  name="min_age"
+                  style={{ fontSize: '1.1rem' }}
+                />
```

Comment:
I believe a .map function can be used to create these three InputNumber fields and clean the code up a little bit.